zha/application/platforms/cover/__init__.py

```diff
@@ -427,7 +427,17 @@ def state(self) -> dict[str, Any]:
         )
         return response
 
-    @property
+    @functools.cached_property
+    def is_opening(self) -> bool | None:
```

Comment:
Actually, for `Cover`, only `bool` is allowed.
`Shade` apparently also allows `None`. Is that needed?

Reply:
Yeah, I think we should remove `None` here.
```suggestion
    def is_opening(self) -> bool:
```